app/dashboard/organization/overview/page.tsx

```diff
@@ -20,17 +21,19 @@ import {
 
 
 type orgDataType = {
-    name: any;
-    email: any;
-    bio: any;
-    website: any;
-    isNonProfit: any;
+    name: string;
+    email: string;
+    bio: string | null;
+    website: string | null;
+    isNonProfit: boolean;
+    owner: string;
+    admins: Array<string>;
 }
 
 export default function OrganizationOverview() {
     const router = useRouter();
     const { toast } = useToast();
-    const [userID, setUserID] = useState("");
+    const [userID, setUserID] = useState(null);
```

Comment:
useStates that have a null value initialized should have a datatype specified for better readability